src/include/rados/librados.h

```diff
@@ -2230,7 +2254,7 @@ CEPH_RADOS_API int rados_aio_stat(rados_ioctx_t io, const char *o,
  * @param o the name of the object to compare with
  * @param completion what to do when the comparison is complete
  * @param cmp_buf buffer containing bytes to be compared with object contents
- * @param cmp_len length to compare and size of @cmp_buf in bytes
```

Comment:
could instead put 
`@c cmp_buf` for rendering `cmp_buf`  in monospace font, see https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdc

Reply:
@tchaikov Did you mean `\c`?

Reply:
please see https://www.stack.nl/~dimitri/doxygen/manual/commands.html

> All commands in the documentation start with a backslash (\) or an at-sign (@). If you prefer you can replace all commands starting with a backslash below by their counterparts that start with an at-sign.